src/mailingui/components/buttons/ReactEmailButtonFork.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
```

Comment:
I get why we need this, but the naming seems pretty weird to me to be honest. @iamhectorsosa is there any good pattern how we can override these library files until we file a proper PRs there?

Reply:
Not sure tbh, I don't have major concerns since this is only for internal consumption.

It is a proprietary implementation though, we aren't importing/consuming anything from React Email. We aren't overriding anything either.

So maybe using a name that alludes to its function, something like "ButtonPrimitive" or "ButtonWrapper", would be better? I'm gonna leave this one to @ericvalcik 

Also could we leave a comment in the component with a TODO to replace this if we are really intending to file a PR to React.email.

```tsx
// If we are expecting to replace this, we can keep the named import as is
import { Button as ReactEmailButton } from "./<FILENAME>";
// To be replaced by
import { Button as ReactEmailButton } from "@react-email/components";
```